data-prepper-core/src/main/java/org/opensearch/dataprepper/pipeline/Pipeline.java

```diff
@@ -211,7 +212,21 @@ public void execute() {
                 final SourceCoordinator sourceCoordinator = sourceCoordinatorFactory.provideSourceCoordinator(partionProgressModelClass, name);
                 ((UsesSourceCoordination) source).setSourceCoordinator(sourceCoordinator);
             }
-            source.start(buffer);
+
+            sinkExecutorService.submit(() -> {
+                long retryCount = 0;
+                while (!isReady() && !isStopRequested()) {
+                    if (++retryCount % 60 == 0) {
+                        LOG.info("Pipeline {} Waiting for Sink to be ready", name);
```

Comment:
A lot of our logs wrap pipeline names in brackets. Let's keep that pattern.

Also, I see a lot of hyphens after this.

So maybe this:

```
LOG.info("Pipeline [{}] - Waiting for Sink to be ready", name);
```